src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -215,18 +212,11 @@
                 object through which to make calls.
               ssl_creds (:class:`grpc.beta.implementations.ClientCredentials`):
                 A `ClientCredentials` for use with an SSL-enabled channel.
-              retrying_override (dict[string, :class:`google.gax.RetryOptions`]): A
-                dictionary that overrides default retrying settings.
-                ``retrying_override`` maps method names (e.g., ``'list_foo'``) to
-                custom RetryOptions objects, or to None. A value of None indicates
-                that the method in question should not retry.
-              bundling_override (dict[string, :class:`google.gax.BundleOptions`]): A
-                dictionary that overrides default bundling settings.
-                ``bundling_override`` maps bundling method names (e.g.,
-                'publish_foo') to custom BundleOptions objects, or to None. It is
-                invalid to have a key for a method that is not bundling-enabled. A
-                value of None indicates that the method in question should not
-                bundle.
+              client_config (dict):
+                A dictionary for call settings for each method. See
```

Comment:
Sorry to go back on what I just said, but can we actually refer to the data structure as "call configuration"? Otherwise I don't like the collision with GAX objects CallOptions/CallSettings